src/include/miopen/solver.hpp

```diff
@@ -731,20 +731,22 @@ struct PerformanceImplicitGemmBwdDataV4R1Xdlops
     int GemmMPerBlock; // 2^n[32..128]
     int GemmKPerBlock; // 2^n[4..16]
 
+    int GemmKPACKSize; // 2^[1..4]
```

Comment:
Existing PerformanceConfigs will become obsolete/corrupt after this PR.

Reply:
@TejashShah Maybe it could be removed to avoid the PerConfig's corrupt? And it will be added back when we make a new config db.